integration/docker/Dockerfile

```diff
@@ -77,18 +76,19 @@ ENV TINI_VERSION v0.18.0
 ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini-static /usr/local/bin/tini
 RUN chmod +x /usr/local/bin/tini
 
-RUN apk add --no-cache openjdk8 openjdk8-jre-lib unzip vim
-
-ENV JAVA_HOME /usr/lib/jvm/java-1.8-openjdk
+RUN \
+    yum install -y java-11-openjdk-devel java-11-openjdk && \
+    yum install -y unzip vim && \
+    yum clean all
 
 # Disable JVM DNS cache (https://github.com/Alluxio/alluxio/pull/9452)
-RUN echo "networkaddress.cache.ttl=0" >> ${JAVA_HOME}/jre/lib/security/java.security
+RUN echo "networkaddress.cache.ttl=0" >> ${JAVA_HOME}/conf/security/java.security
 
 # Add the following for native libraries needed by rocksdb
 ENV LD_LIBRARY_PATH /lib64:${LD_LIBRARY_PATH}
```

Comment:
There are two `ENV LD_LIBRARY_PATH` on line 65 and 88. Let's combine them. The `/usr/local/lib` is for libfuse2 - let's put this information in the comment. People wouldn't know what this is for (for example, me a few months ago)